### ISSUE TITLE
Use /proc instead of free -m

### ISSUE THREAD

**Issue by 11mariom** [sensu-plugins/sensu-plugins-memory-checks#14]
Hi,

there is one serious problem with check-ram.rb:

``` ruby
`free -m`.split("\n").drop(1).each do |line|
free_ram = line.split[3].to_i if line =~ /^-\/\+ buffers\/cache:/ # rubocop:disable RegexpLiteral
```

It is not best way to relate on free -m. Why? Output can be different on different distributions. f.e.
Centos 6:

``` bash
[root@centos6]# free -m
             total       used       free     shared    buffers     cached
Mem:          8192       1191       7000          2          0        953
-/+ buffers/cache:        237       7954
Swap:         4096          0       4096
```

Centos 7:

``` bash
[root@centos7]# free -m
              total        used        free      shared  buff/cache   available
Mem:           8192         429         956           3        6806        7633
Swap:          4096         198        3897
```

This is causing false-positives in monitoring, because calling script gives output:

``` bash
[root@centos7]# /opt/sensu/embedded/bin/check-ram.rb
CheckRAM CRITICAL: 0% free RAM left
```

I think it will be better to use /proc/meminfo - there is very small chance to change of file format.

**Comment by mattyjones** [comment]
@11mariom  I would agree with this without question.  I would like to look at using vmstat though as it is cross-platform.  The more pure ruby we can use the better.
